Tests/SwaggerParserTests/SwaggerParserTests.swift

```diff
@@ -1,20 +1,129 @@
-
 class SwaggerParserTests: XCTestCase {
-    func testUber() {
-        do {
-            let swagger = try Swagger(JSONString: uberJSONString)
-            print(swagger)
-        } catch {
-            print(error)
+    var testFixtureFolder: URL!
+    
+    override func setUp() {
+        testFixtureFolder = URL(fileURLWithPath: #file).deletingLastPathComponent().appendingPathComponent("Fixtures")
+    }
+    
+    func testInitialization() {
+        let url = testFixtureFolder.appendingPathComponent("uber.json")
+        let jsonString = try! NSString(contentsOfFile: url.path, encoding: String.Encoding.utf8.rawValue) as String
+        let swagger = try! Swagger(JSONString: jsonString)
+        
+        XCTAssertEqual(swagger.host?.absoluteString, "api.uber.com")
+    }
+    
+    func testAllOfSupport() {
+        let url = testFixtureFolder.appendingPathComponent("test_all_of.json")
+        let jsonString = try! NSString(contentsOfFile: url.path, encoding: String.Encoding.utf8.rawValue) as String
+        let swagger = try! Swagger(JSONString: jsonString)
+        
+        guard
+            let baseDefinition = swagger.definitions.first(where: {$0.name == "TestAllOfBase"}),
+            case .object(let baseSchema) = baseDefinition.structure else {
+                XCTFail("TestAllOfBase is not an object schema."); return
         }
+        
+        SwaggerParserTests.validate(testAllOfBaseSchema: baseSchema)
+        swagger.definitions.validate(testAllOfChild: "TestAllOfFoo", withPropertyNames: ["foo"])
```

Comment:
This test was intended to validate that the swagger.json file is being parsed correctly which implies that each object should be validated within this test. Should I break it out into a separate test per schema being parsed? 

If we do break them out, we may want to consider making separate test files for each feature. 

I feel like a BDD framework would make breaking things out more granularly easier to manage (with `context`s and `describe` scopes).